src/PureCloud.Client/Models/VoicemailMailboxInfoClean.cs

```diff
@@ -0,0 +1,15 @@
+namespace PureCloud.Client.Models;
+
+public sealed class VoicemailMailboxInfoClean
```

Comment:
Don't prefix the file name with the word "Clean". Used the same file name as the existing model.